Sources/AppStoreConnectCLI/Services/Operations/ListBuildsOperation.swift

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct ListBuildsOperation: APIOperation {
+
+    struct Options {
+        let filterAppIds: [String]
+        let filterExpired: [String]
+        let filterPreReleaseVersions: [String]
+        let filterBuildNumbers: [String]
+        let filterProcessingStates: [ListBuilds.Filter.ProcessingState]
+        let filterBetaReviewStates: [String]
+        let limit: Int?
+    }
+
+    typealias Build = AppStoreConnect_Swift_SDK.Build
+    typealias Relationships = [AppStoreConnect_Swift_SDK.BuildRelationship]?
+
+    typealias Output  = [(build: Build, relationships: Relationships)]
+
+    private let options: Options
+
+    init(options: Options) {
+        self.options = options
+    }
+
+    func execute(with requestor: EndpointRequestor) -> AnyPublisher<Output, Error> {
+        var filters = [ListBuilds.Filter]()
+        filters += options.filterAppIds.isEmpty ? [] : [ListBuilds.Filter.app(options.filterAppIds)]
```

Comment:
Relying on type inference you could choose to omit all the type prefixes here:
```suggestion
        filters += options.filterAppIds.isEmpty ? [] : [.app(options.filterAppIds)]
```